core/src/main/java/com/linecorp/armeria/server/docs/DocServiceBuilder.java

```diff
@@ -448,6 +452,20 @@ public DocServiceBuilder injectedScriptSupplier(
         return this;
     }
 
+    /**
+     * Adds the specified {@link NamedTypeInfoProvider}s used to create a {@link NamedTypeInfo} from
+     * a type descriptor.
+     */
+    public DocServiceBuilder namedTypeInfoProvider(NamedTypeInfoProvider namedTypeInfoProvider) {
+        requireNonNull(namedTypeInfoProvider, "namedTypeInfoProvider");
+        if (this.namedTypeInfoProvider == null) {
+            this.namedTypeInfoProvider = namedTypeInfoProvider;
+        } else {
+            this.namedTypeInfoProvider.orElse(namedTypeInfoProvider);
```

Comment:
I'm not sure which one is better for users' experience. 
If users want to prioritize multiple `NamedTypeInfoProvider`s, they should manually compose the `NamedTypeInfoProvider`s using `orElse` rather than, relying on how the method prioritizes the injected `NamedTypeInfoProvider`.